include/OperatorsForMesons.hpp

```diff
@@ -47,6 +48,16 @@ class OperatorFactory {
                   const std::string &path_vdaggerv,
                   const std::string &path_config,
                   const HypPars &hyp_parameters);
+  OperatorFactory(const ssize_t Lt,
```

Comment:
Why not just call the constructor with ```dilE = 0``` ? 

I was rather thinking that one would move ```build_vdaggerv``` into a free function if possible and then simply call that from the ```VdaggerV``` executable when running a pure VdaggerV job or  from ```OperatorFactory``` when running as part of a contraction run.

Certainly this duplication of the constructor is not necessary.